code/threshold_canny.ipynb

```diff
@@ -0,0 +1,235 @@
+{
```

Comment:
<p>"we should test on other images that this threshold also works this well. " Completely agree. I will talk with @<a href="https://github.com/ralph-melcher" target="_blank">ralph-melcher </a>to get more material. </p><p> </p><p>About the threshold:</p><p> </p><p>Would it make sense to use a function that determines a useful threshold? Given that the color values of Liver pixels, and Fat pixels are very close, one could compare them and try to cancel out only the Fat pixels.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/markusait/lcmd/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>